AdminUi/src/AdminUi/ClientApp/src/app/services/logger-server-service/logger-server.service.ts

```diff
@@ -27,11 +27,16 @@ export class LoggerServerService extends NGXLoggerServerService {
         return httpRequest;
     }
 
-    public override customiseRequestBody(metadata: INGXLoggerMetadata) {
+    protected override customiseRequestBody(metadata: INGXLoggerMetadata) {
+        let messageTemplate: string = metadata.message;
+        for (let i = 0; i < metadata.additional!.length; i++) {
+            messageTemplate = messageTemplate.replace("%s", `{${i}}`);
```

Comment:
What if I want to actually log "%s"? Or what if I use "%d" for example?

You should therefore use the following regex:

```
(?<!%)%[dsj]
```